Update raut2webstr-pagemodel-tree.py

ltrilety opened the pull request:
Fix __init__.py file name
Add some imports to the __init__.py file

ltrilety committed:
Update raut2webstr-pagemodel-tree.py

Fix __init__.py file name
Add some imports to the __init__.py file

ltrilety committed:
Fix import_path creation

change evaluation of import_path when no path is provided to move_files method

mbukatov committed:
fix index numbers in format strings

mbukatov reviewed (changes requested):
There are few issues with content of `__init__.py` files.

ltrilety committed:
Fix module names

ltrilety committed:
Improve work with root_path

If root path to the project is not provided, don't put any import to the __init__.py file

mbukatov commented:
Looks good, thanks.
